src/main/java/app/controllers/TimeZonesController.java

```diff
@@ -15,4 +15,4 @@ private static void index(Context ctx, ConnectionPool connectionPool)
     {
         ctx.render("/timezones/index.html");
     }
-}
```

Comment:
why this change?

Reply:
Jeg har fået det rettet inde på readme, ved ikke om det er rigtig gjort?

Reply:
Kan du se ændring?